modules/ROOT/pages/policy-mule4-detokenization.adoc

```diff
@@ -21,30 +21,41 @@ endif::[]
 
 |===
 
-[NOTE]
---
-This policy is designed to work only for Anypoint Runtime Fabric deployments.
---
+Detokenization is the process of transforming tokenized data back into its original value to redeem sensitive data for fulfilling specific business needs. You can tokenization your data to replace sensitive information with tokens to reduce the risk of compromising sensitive information. 
 
-== Detokenization
-
-Detokenization is the process of detokenizing a token back into its original value to redeem sensitive data to carry on with a specific business purpose.
-For example, an application would need to use a detokenized value to validate a specific credit card number. +
-Detokenized sensitive data must always be read under strict security controls.
+For example, an application can require the bank account number or the credit card number for generating monthly credit card statements. Detokenized sensitive data must always be read under strict security controls.
 
 == Prerequisites
 
-* To create and apply policies you need the *Manage Policies* permission.
-* You need to have a tokenization service up and running.
-** To configure and see the tokenization service, you need the *Manage Tokenization Services* permission. +
-See xref:anypoint-security::tokenization-example.adoc[Configure and Use Tokenization] for more information.
+To apply and manage the Detokenization policy, you must have:
+
+* Obtained the *Manage Policies* permission.
+* To xref:anypoint-security::tokenization-example.adoc[configure and use the tokenization service], you must have:
+** Installed Runtime Fabric
+** Obtained the *Manage Tokenization Services* permission
+* An operating tokenization service
+
+== Scope and Considerations
+
+Before you apply this policy, review the following considerations:
 
+* One instance of the policy supports only one tokenization service. 
++
+All the formats that you configured need to be contained in the same tokenization service.
+* You can use one instance of the policy to tokenize a value either in the request or the response, but not in both.
+* The policy is not aware of the state of the tokenization service until runtime. If you deleted your tokenization service or if it is not operational, the policy cannot establish communication with it. This causes all the requests to fail until the service is available again.
+* During policy configuration, the services menu displays only the tokenization services that are ready to use. 
++
+If you do not see the service you want, check if it is already deployed.
```

Comment:
```suggestion
If you do not see the service you want, check to see if it is already deployed.
```
Should we add "...by <doing this>" to tell them how?

Reply:
Thanks!